style-rtl.css

```diff
@@ -964,6 +967,25 @@ textarea:focus {
   outline-offset: -4px;
 }
 
+input[type="text"]::-webkit-search-decoration,
```

Comment:
One more small change: This `&::-webkit-search-decoration` property can be separated out so its specific to `input[type="search"]`. Since that property doesn't apply to non-search input fields, most of these auto-compiled styles aren't necessary. 

Thanks again!

Reply:
Sorry for forgetting that point.
I fixed it again!

Reply:
Looks great, thank you @youthkee!